resources/views/home.blade.php

```diff
@@ -101,7 +101,7 @@ class="px-8 py-6 font-sans text-xl font-semibold text-white transition duration-
                 <div class="relative w-full px-8 pt-12 pb-24 mb-10 font-sans bg-white rounded-lg shadow lg:w-1/3 lg:first:mr-8 lg:last:ml-8 last:mb-0 lg:mb-0">
                     <div class="mb-12 text-2xl text-center text-indigo-800">{{ $conference->title }}</div>
                     <div class="text-xl font-medium">{{ $conference->event_dates_display }}</div>
-                    <div class="mt-6 text-xl">{{ $conference->description }}</div>
+                    <div class="mt-6 text-xl">{{ mb_substr($conference->description, 0, 100) }}</div>
```

Comment:
Perhaps we can use [Laravel's string `limit` function](https://laravel.com/docs/9.x/helpers#method-str-limit) here so that we can get the sweet ellipses at the end of the excerpt.

Reply:
Good call!  I just updated this here and on the conference listing page.